### DISCUSSION TITLE
vscode-webview-ui-toolkit validation

### DISCUSSION THREAD

**Discussion by markwallace-microsoft** [microsoft/vscode-discussions#655] [Extension Development QnA] [answered]
Hi,

I'm using the vscode-webview-ui-toolkit to build a couple of web views for our extension. I need to add some custom validation e.g., ensure users only enter numbers between -2 & +2 for certain fields.

Is there a validation library that works well with the webview ui toolkit and provides a good experience for  the users of a VS Code extension?

Thanks
Mark

**Comment by isidorn** [comment, marked as answer]
I am not aware of a validation library. But @daviddossett might know something

**Reply by daviddossett** [reply]
Unfortunately I'm not aware of anything that would specifically be a good option for the toolkit. cc @hawkticehurst
